.rubocop.yml

```diff
@@ -142,7 +142,7 @@ RSpec:
       - expect_offense
 
 RSpec/PredicateMatcher:
-  EnforcedStyle: explicit
+  Strict: false
```

Comment:
I'd add a comment explaining the need for this.

Reply:
Added:
> Also consider `be(true)` and `be(false)` as offenses, instead of only `be_truthy` and `be_falsy`